app/Models/ContactInfo.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class ContactInfo extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'phone_number',
+        'email',
+        'gender',
+    ];
+
+    public $timestamps = false;
+
+    public function customer()
+    {
+        return $this->belongsTo('Customer');
```

Comment:
please specify full classpath `Customer::class`, don't forget about the import

Reply:
here you should replace relation with direct `hasOne`
https://laravel.com/docs/8.x/eloquent-relationships#one-to-one

Reply:
Added class path, and fixed the relation